harvester/pmh_interface.py

```diff
@@ -47,16 +49,27 @@ def fetch_mets(self, dc_identifier, folder_path, file_name=None):
         xml_response = requests.get(mets_url, timeout=5)
         xml_response.raise_for_status()
 
+        if not folder_path:
+            folder_path = self._default_mets_path()
+
         if not file_name:
-            path = f"{folder_path}/{utils.binding_id_from_dc(dc_identifier)}_METS.xml"
-        else:
-            path = f"{folder_path}/{file_name}"
+            file_name = f"{utils.binding_id_from_dc(dc_identifier)}_METS.xml"
+
+        folder_path = Path(folder_path)
+        folder_path.mkdir(parents=True, exist_ok=True)
+        path = str(folder_path / file_name)
```

Comment:
We actually don't need to do this conversion to string: `Path`s are "path-like" so `open` can digest them as-is.

Reply:
I remember there being a reason for doing this, but I no longer remember what it is and the code works fine without the conversion. Fixing!